tailwind.config.js

```diff
@@ -83,11 +84,6 @@ module.exports = {
         '3xl': '1600px',
         '4xl': '2000px',
       },
-      boxShadow: {
```

Comment:
Was this boxShadow removed on purpose? Just seems unrelated to the plugins lol

Reply:
Yes! It was the utility that added the animated background before the fix!